.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,58 @@
+name: DualGovernance PR tests
+
+permissions:
+  contents: read
+
+on:
+  pull_request:
+    branches: ['main', 'develop']
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+
+    strategy:
+      matrix:
+        include:
+          - id: regressions
+            display: "regressions"
+            command: "npm run test:regressions -- --load-accounts"
```

Comment:
Since the tests are executed in parallel and the load accounts are called in one of them, some tests may run with outdated stETH balances. Consider loading the accounts once before running all tests.